Unreal/CarlaUE4/Plugins/CarlaTools/Source/CarlaTools/Private/HoudiniImporterWidget.cpp

```diff
@@ -72,7 +73,7 @@ void UHoudiniImporterWidget::MoveActorsToSubLevelWithLargeMap(TArray<AActor*> Ac
       continue;
     }
 
-    UWorld* World = UEditorLevelLibrary::GetEditorWorld();
+    UWorld* World = GEditor->GetEditorWorldContext().World();
```

Comment:
**Code Clean***
The next code should be also removed from this file as this include is not used anymore:
```c++
#include "EditorLevelLibrary.h"
```